lib/tasks/demo.rake

```diff
@@ -423,6 +423,23 @@ namespace :demo do
       patient[:pause_notifications] = rand < 0.1
       patient[:last_assessment_reminder_sent] = beginning_of_day - rand(7).days if rand < 0.3
 
+      # Follow-up Flag
+      if rand < 0.15
+        follow_up_reasons = [
```

Comment:
can we define this in the validation helper (pretty much exactly how the `monitoring_reason` field is populated)

Reply:
If we're going to define these in the validation helper, then should they also be passed to the react component to generate the dropdown options?

Reply:
I like that idea too

Reply:
That is how I originally implememnted it, but per [Sarah's suggestion](https://github.com/SaraAlert/SaraAlert/pull/899#discussion_r634529951), the flagged reasons were refactored to be included inline. I agreed with her suggestion because it seemed unecessary to pass that `prop` between all the various components for it to just be used by the `FollowUpFlag` component in the end.

Reply:
for reference, the reason I suggested this is because it's only used in one spot in the front end, where as something line `monitoring_reason` is used in quite a few

Reply:
I'm ok with this not being passed as props to the front end, but would definitely like this to be defined in the validation helper instead of being redefined as a local variable within the loop that creates patients

Reply:
@timwongj I can certainly look to separate this array into the validation helper. I'll also add a comment just above this array in the validation helper noting that the follow up reasons are also rendered in-line in the `FollowUpFlag`  component.